op-batcher/batcher/driver.go

```diff
@@ -287,7 +287,7 @@ func (l *BatchSubmitter) loadBlocksIntoState(ctx context.Context, start, end uin
 			// This allows the batcher to start publishing sooner in the
 			// case of a large backlog of blocks to load.
 			trySignal(publishSignal)
-			l.sendToThrottlingLoop(pendingBytesUpdated)
+			l.sendToThrottlingLoop(unsafeBytesUpdated)
```

Comment:
This is a non-blocking send which drops the signal if the go channel is full. This could mean the metric is not updated, if say there are a lot of blocks being loaded (e.g. after a restart). 

We should think about a better pattern to keep the metric updated at all times.

Reply:
Throttler could check every `PollInterval`. That also separates the concerns a bit more: no more passing around channels.